guide/getting-started/visual-language.md

```diff
@@ -203,6 +223,35 @@ Think about Bitcoin’s visual language as a scale that ranges from muted to hig
 
 In digital product design, there are times when a minimally expressive approach is what’s needed. For example, when a user makes a transaction, we want them to focus. At other times, illustrations and stronger visual elements are more appropriate, and this is where we can take advantage of the creative freedom Bitcoin affords.
 
+## The Lightning network
+
+The Lightning network is a layer 2 solution for scaling Bitcoin. It allows people to make almost instant payments with very small fees.
+
+Since the first Lightning implementation in 2018, a distinct visual identity has developed around the Lightning network based on a simple yellow icon of a lightning bolt. Sometimes, Lightning software and services may choose to completely forego common elements of Bitcoin branding (the color orange, the Bitcoin symbol, etc.) and instead use the color yellow, a lightning bolt symbol, and metaphors for electricity.
+
+<div class="image-gallery" markdown="1">
+
+{% include image.html
+image = "/assets/images/guide/getting-started/visual-language/bitcoin-lightning-branding.jpg"
+retina = "/assets/images/guide/getting-started/visual-language/bitcoin-lightning-branding@2x.jpg"
+alt-text = "Sequence of images showing examples of logos and brands for lightning software and services"
+caption = "References from top top bottom, left to right: <a href='https://www.walletofsatoshi.com/' target='_blank'>1</a>, <a href='https://shop.fulmo.org/raspiblitz/' target='_blank'>2</a>, <a href='https://www.lightningnetworkstores.com/' target='_blank'>3</a>, <a href='https://zeusln.app/' target='_blank'>4</a>, and <a href='https://lightning.store/product/bitcoin-rtl-t-shirt/' target='_blank'>5</a>."
+width = 800
+height = 600
+%}
+
+</div>
+
+## Should Lightning be distinct from Bitcoin?
+
+While Lightning branding can be fun and expressive, it may give new Bitcoin users the impression that Lightning is different from Bitcoin. More accurately, Lightning should be thought of as a different way of using bitcoin, not as a different currency than bitcoin.
+
+#### Consider your audience
+
+Are you developing Bitcoin products that are intended to be used primarily by developers or people who are very experienced with Bitcoin? If so, lightning branding may be a great choice for your product.
```

Comment:
I think you brought this up jokingly, but I am not confused about the mixed use of lowercase and uppercase Lightning/lightning. "Consider your audience" is lowercase, which seems to break with the rest of the page.

Reply:
That's just a typo. I'll correct the casing throughout.

Reply:
Thank you.

Reply:
Resolved in eb50035